### ISSUE TITLE
/sbin/ipfw: No such file or directory on MacOS 10.13

### ISSUE THREAD

**Issue by TheTomCanuck** [zquestz/throttled#1]
Just found out about throttled but appears it doesn't work on newer MacOS?

mac:throttled tom$ sudo ./throttled-startup 
iTunes TTL fix has been enabled!
Max bytes/sec has been set to 16384
Speed change set to 1024 bytes
Priority has been set to -19
Creating a socket for divert port 17779
Creating a socket for divert port 17778
Creating a socket for divert port 17777
Waiting for data...
./throttled-startup: line 61: /sbin/ipfw: No such file or directory
./throttled-startup: line 62: /sbin/ipfw: No such file or directory
./throttled-startup: line 65: /sbin/ipfw: No such file or directory
./throttled-startup: line 68: /sbin/ipfw: No such file or directory
./throttled-startup: line 69: /sbin/ipfw: No such file or directory
./throttled-startup: line 72: /sbin/ipfw: No such file or directory
./throttled-startup: line 81: /sbin/ipfw: No such file or directory
./throttled-startup: line 82: /sbin/ipfw: No such file or directory
./throttled-startup: line 85: /sbin/ipfw: No such file or directory
./throttled-startup: line 86: /sbin/ipfw: No such file or directory
./throttled-startup: line 89: /sbin/ipfw: No such file or directory
./throttled-startup: line 90: /sbin/ipfw: No such file or directory
./throttled-startup: line 91: /sbin/ipfw: No such file or directory
./throttled-startup: line 94: /sbin/ipfw: No such file or directory
./throttled-startup: line 97: /sbin/ipfw: No such file or directory
./throttled-startup: line 103: /sbin/ipfw: No such file or directory
./throttled-startup: line 106: /sbin/ipfw: No such file or directory
./throttled-startup: line 107: /sbin/ipfw: No such file or directory
./throttled-startup: line 108: /sbin/ipfw: No such file or directory
./throttled-startup: line 111: /sbin/ipfw: No such file or directory
./throttled-startup: line 112: /sbin/ipfw: No such file or directory
./throttled-startup: line 113: /sbin/ipfw: No such file or directory
./throttled-startup: line 121: /sbin/ipfw: No such file or directory
./throttled-startup: line 122: /sbin/ipfw: No such file or directory
./throttled-startup: line 125: /sbin/ipfw: No such file or directory
./throttled-startup: line 128: /sbin/ipfw: No such file or directory
./throttled-startup: line 129: /sbin/ipfw: No such file or directory
./throttled-startup: line 132: /sbin/ipfw: No such file or directory
./throttled-startup: line 135: /sbin/ipfw: No such file or directory
./throttled-startup: line 136: /sbin/ipfw: No such file or directory
./throttled-startup: line 137: /sbin/ipfw: No such file or directory
./throttled-startup: line 140: /sbin/ipfw: No such file or directory
./throttled-startup: line 141: /sbin/ipfw: No such file or directory
./throttled-startup: line 142: /sbin/ipfw: No such file or directory
./throttled-startup: line 145: /sbin/ipfw: No such file or directory
./throttled-startup: line 146: /sbin/ipfw: No such file or directory
./throttled-startup: line 147: /sbin/ipfw: No such file or directory
./throttled-startup: line 148: /sbin/ipfw: No such file or directory
./throttled-startup: line 151: /sbin/ipfw: No such file or directory
./throttled-startup: line 154: /sbin/ipfw: No such file or directory
./throttled-startup: line 155: /sbin/ipfw: No such file or directory
./throttled-startup: line 158: /sbin/ipfw: No such file or directory

**Comment by zquestz** [comment]
Yep unfortunately Apple removed ipfw from recent versions of Mac OS X, and does not have a reasonable way to divert packets to the throttled daemon any longer. =(
